app/src/main/java/org/mifos/mobile/ui/fragments/ClientAccountsFragment.kt

```diff
@@ -339,6 +328,9 @@ import javax.inject.Inject
                     checkBoxAdapter?.statusList = (childFragmentManager
                             .findFragmentByTag(getFragmentTag(0)) as AccountsFragment?)?.getCurrentFilterList()
                 }
+                if(resources.configuration.orientation == Configuration.ORIENTATION_LANDSCAPE){
+                    checkBoxRecyclerView!!.layoutParams.height = (getScreenHeight()/3.5).toInt()
+                }
```

Comment:
Is there any reason why you have copy pasted the same part of the code in 3 different places, what about if you remove this part of the code from the `when` block and maybe put it under line number 321? This way, you won't need to replicate it.

Reply:
You are right, we can write that at 321 and the if condition would be checked once but even if we write it 3 times even in this case the if condition would be executed once because at a time only one the three account types would be clicked. 
Having said that , I can fix this and push it in the next commit if you would say so

Reply:
done !!

Reply:
And the last change, can we use `?.` safe call operator instead of `!!`?

Reply:
done @jawidMuhammadi